### PR TITLE
Feature/loading spinner

### PR DESCRIPTION
**Pull request by LudvigLundberg**
Added a VERY simple (and ugly) spinner when checkForCopper is dispatched, that disappears when the fetch has been answered. 
Also added a spinner in front of the loading-button that activates when user activates a button. 

KNOWN ISSUE:
When the user gets redirected back to the website, a new App component is created, this causes problems with the loading spinner, since the state of the new App is "loadingUser: false" by default.

### CODE REVIEW

**Review by MoaEkberg — COMMENTED** [review]
The loading spinner shows nicely at login to start with. But then page reloads and we get to loginpage once again , before getting into the game, and the spinner is now gone as if it failed to login.

When the spinner shows its very obvious tho what's going on.

**Comment by nij0** [comment]
This PR looks good to me! There has been an issue opened about the bug [#113](https://github.com/pvtworld/pvt/issues/113) that can be solved in the next coming sprint.

**Comment by 2ECC71** [comment]
Could we change login method to popup? Works on mobile for me.
If we do, the spinner work as expected.

**Comment by Murmellow** [comment]
Can someone please enlighten me why this is still open? Is it rejected waiting refactoring or simply has yet to be reviewed? Comments seem to show that it has been reviewed but not authorised to be merged?

**Comment by nij0** [comment]
as @loxtank mentioned this solution will work just fine with a popup instead of redirect. I guess we're maybe waiting for that to happen? :)

File: src/Components/App/App.js [hunk 1]
```diff
@@ -43,7 +45,8 @@ export default class App extends React.Component {
     }
 
     authenticate(provider) {
-        console.log(`Trying to log in with ${provider}`);
+        console.log(`Trying to log in with ${provider}`); 
+        this.setState({userLoading: true})
         base.authWithOAuthRedirect(provider, this.authHandler);
```
Comment (2ECC71) [inline src/Components/App/App.js]: should we change this to 
`base.authWithOAuthPopup(provider, this.authHandler);`
?

**Review by 2ECC71 — CHANGES_REQUESTED** [review]
I think this looks good, let's merge it after a change to popup

File: src/Components/App/App.js (hunk 1 above)
Comment (nij0) [reply src/Components/App/App.js]: LGTM

**Review by nij0 — APPROVED** [review]
LGTM

**Review by nij0 — APPROVED** [review]
LGTM

**Review by nij0 — APPROVED** [review]
LGTM

**Review by 2ECC71 — APPROVED** [review]

**Pull request merged**